maskrcnn_benchmark/structures/segmentation_mask.py

```diff
@@ -90,7 +136,8 @@ def transpose(self, method):
         return Polygons(flipped_polygons, size=self.size, mode=self.mode)
 
     def crop(self, box):
-        w, h = box[2] - box[0], box[3] - box[1]
+        TO_REMOVE = 1
+        w, h = box[2] - box[0] + TO_REMOVE, box[3] - box[1] + TO_REMOVE
```

Comment:
Indeed, this used to look inconsistent, but I think [it mimics the behavior of Detectron in this part of the code](https://github.com/facebookresearch/Detectron/blob/master/detectron/utils/segms.py#L110), so I'd maybe rather keep a comment in here instead of modifying the code, as it will change the accuracies (for the better or for the worse)?

Reply:
Yes, this is supposed to be the conversion from one format to the other, but there is an inconsistency in Detectron I believe, and I tried to keep this inconsistency here for the sake of reproducibility

Reply:
Could you please point me out where the inconsistency happens in Detectron?

Reply:
In detectron, they use the +1 everywhere, except in the masks in the point I mentioned to you. Have a look at my other comment for a bit more info